app/src/main/java/org/thoughtcrime/securesms/conversation/v2/ConversationAdapter.kt

```diff
@@ -40,9 +40,8 @@ class ConversationAdapter(
     private val onItemLongPress: (MessageRecord, Int, VisibleMessageView) -> Unit,
     private val onDeselect: (MessageRecord, Int) -> Unit,
     private val glide: GlideRequests,
-    lifecycleCoroutineScope: LifecycleCoroutineScope
-)
-    : CursorRecyclerViewAdapter<ViewHolder>(context, cursor) {
+    private val lifecycleCoroutineScope: LifecycleCoroutineScope
```

Comment:
@hjubb Since I'm not too familiar with it I just wanted to check that storing the `LifecycleCoroutineScope` in a variable won't cause memory leaks?

Reply:
No worries, as per your other suggestion I've changed this to take a closure instead which seems a bit cleaner (since it has to be passed through multiple files)